frontend/src/store/index.js

```diff
@@ -165,6 +168,47 @@ const isValidRegion = (getters, cloudProfileName, cloudProviderKind) => {
   }
 }
 
+function mapOption (optionValue, shootResource) {
+  const key = get(optionValue, 'key')
+  if (!key) {
+    return
+  }
+
+  const isSelectedByDefault = get(optionValue, 'isSelectedByDefault', false)
+  const inputInverted = get(optionValue, 'input.inverted', false)
+  const defaultValue = inputInverted ? !isSelectedByDefault : isSelectedByDefault
+  const rawValue = get(shootResource, ['metadata', 'annotations', key], `${defaultValue}`) === 'true'
+  const value = inputInverted ? !rawValue : rawValue
+
+  const option = {
+    value
+  }
+  return [key, option]
+}
+
+function mapAccessRestriction (accessRestrictionValue, shootResource) {
```

Comment:
..or rather test this method which will test the other one, too

Reply:
✅